lib/public/components/Filters/RunsFilter/runTypeFilter.js

```diff
@@ -11,46 +11,17 @@
  * or submit itself to any jurisdiction.
  */
 
-import { picker } from '../../common/selection/picker/picker.js';
-import { h } from '/js/src/index.js';
+import { selectionDropdown } from '../../common/selection/dropdown/selectionDropdown.js';
+import { combinationOperatorChoiceComponent } from '../common/combinationOperatorChoiceComponent.js';
 
 /**
- * Returns a filter component to filter on environment Ids, either a coma separated list of specific ids or a substring
+ * Returns a filter component to filter on environment Ids, either a comma separated list of specific ids or a substring
```

Comment:
This was a wrong copy paste in the previous code, could you please update the comment?